lib/travis/model/repository/settings.rb

```diff
@@ -10,6 +10,7 @@ class EnvVar < Travis::Settings::Model
     attribute :name, String
     attribute :value, Travis::Settings::EncryptedValue
     attribute :public, Boolean, default: false
+    attribute :branch, String, default: ''
```

Comment:
@piotr-travisci i am wondering if this should have a default. would that mean that saving this would result in an empty string in the db, rather than `null`?

Reply:
@svenfuchs 
Thank you for you suggestion. I changed default value for branch.